auth/src/main/res/values/colors.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <color name="fui_colorPrimary">#3F51B5</color>
-    <color name="fui_colorPrimaryDark">#303F9F</color>
-    <color name="fui_colorAccent">#FF4081</color>
+<resources xmlns:tools="http://schemas.android.com/tools">
+    <color name="colorPrimary" tools:ignore="ResourceName">#3F51B5</color>
```

Comment:
Oops.  @TheCraftKid tried to stop me from doing this and I told him it would be fine.

Does this come into play if the app including FirebaseUI does not use AppCompat at all so `colorPrimary` is undefined?